test/Interop/Cxx/class/mutability-annotations.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop)
```

Comment:
To be honest, I don't think this really tests anything. If this were imported as a non-mutating function, the test would still pass. 

I think the module interface test is probably sufficient, but if you want something else, you could do a type checker test with `-verify` to make sure that you get an error why calling it on a `let` constant.

Reply:
Yeah, you're right, I added a typechecker test instead of this one.